doc/installing.md

```diff
@@ -16,16 +16,18 @@ docker run -it --rm -p 3000:3000 prairielearn/prairielearn
 
 * Step 4: When you are finished with PrairieLearn, type Control-C on the commandline where your ran the server to stop it.
 
-* Step 5: To use your own course, point Docker to the correct directory (replace the precise path with your own) on Windows:
+* Step 5: To use your own courses, use the `-v` flag to point your own course directories to the Docker `/course` directories. There are nine available mount points in the Docker: `/course`, `/course2`, `/course3`, ..., `/course9`.
```

Comment:
How about mentioning the 9 mount points outside of instructions for just loading a course? 

Many will copy and paste these instructions... I'm worried that showing 2x or more courses is unneeded in this step.

Reply:
Makes sense, done. Also slightly reworded the single mount instructions for clarity (I think), but I can revert to original if preferred.